src/chains/ethereum_1/ethErrors.ts

```diff
@@ -11,7 +11,7 @@ export const ChainErrorRegExs: { [key: string]: string } = {
 }
 
 // Maps an Error object (thrown by a call to the chain) into a known set of errors
-// RpcError is an eosjs structure that includes the 'json' property that has error details
+// RpcError is an web3js structure that includes the 'json' property that has error details
```

Comment:
This comment is not correct. The Rpcerror object is something from eosjs and isn't in web3.
Also, we should visit how errors are working from web3 - I suspect we need to change something here